corehq/warehouse/transforms/sql/app_status_synclog_staging.sql

```diff
@@ -0,0 +1,39 @@
+INSERT INTO {{ app_status_synclog_staging }} (
+       last_sync,
+       sync_build_version,
+       user_id,
+       domain,
+       batch_id
+)
+SELECT
+    last_sync,
+    sync_build_version,
+    user_id,
+    domain,
+    '{{ batch_id }}'
+FROM
+(
+SELECT
+	row_number() over wnd,
+	CASE
+	    WHEN sync_date < app_status.last_sync_log_date
+	    THEN app_status.last_sync_log_date
+	    ELSE first_value(sync_date AT TIME ZONE 'UTC') OVER wnd
+	END AS last_sync,
+	CASE
+	WHEN sync_date < app_status.last_sync_log_date
+	THEN app_status.last_sync_log_app_build_version
+        ELSE app_dim.version::VARCHAR
+	END as sync_build_version,
+	sync_staging.user_id as user_id,
+	sync_staging.domain as domain
+        FROM {{ synclog_staging }} as sync_staging
+	LEFT JOIN {{ app_status_fact }} as app_status
```

Comment:
i think i see why app status fact is listed as a dependency above now. just fyi that it creates a circular dependency in the dependency list

Reply:
We do not actually use the depencies in hq anywhere except adding the table slugs to the sql template context. In airflow, where we handle the actual dependency management, these tables do not depend on the app status fact since we update these first.